contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -941,7 +941,7 @@ get_authid_user_ext_db_users(const char *db_name)
 char *
 get_user_for_database(const char *db_name)
 {
-	const char		*user = NULL;
+	char		*user = NULL;
```

Comment:
Whats the reason to remove the const?
I see that we are later casting from (const char*) to (char*) which should be avoided

Reply:
Because function returns user whose return type is char*.